Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.15/x64/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.9.15/x64/lib/python3.9/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.9.15/x64/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.9.15/x64/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.9.15/x64/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.9.15/x64/lib/python3.9/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.9.15/x64/lib/python3.9/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.9.15/x64/lib/python3.9/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from jax import numpy as jnp
from jax import random as jr
from matplotlib import pyplot as plt

from dynamax.plotting import  plot_uncertainty_ellipses
from dynamax.nonlinear_gaussian_ssm.nonlinear_gaussian_ssm import ParamsNLGSSM, NonlinearGaussianSSM
from dynamax.nonlinear_gaussian_ssm.extended_kalman_filter import extended_kalman_filter as ekf
from dynamax.nonlinear_gaussian_ssm.unscented_kalman_filter import unscented_kalman_filter as ukf
from dynamax.nonlinear_gaussian_ssm.unscented_kalman_filter import UKFHyperParams

------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [0;32mIn [2], line 5[0m
[1;32m      2[0m [38;5;28;01mfrom[39;00m [38;5;21;01mjax[39;00m [38;5;28;01mimport[39;00m random [38;5;28;01mas[39;00m jr
[1;32m      3[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmatplotlib[39;00m [38;5;28;01mimport[39;00m pyplot [38;5;28;01mas[39;00m plt
[0;32m----> 5[0m [38;5;28;01mfrom[39;00m [38;5;21;01mdynamax[39;00m[38;5;21;01m.[39;00m[38;5;21;01mplotting[39;00m [38;5;28;01mimport[39;00m  plot_uncertainty_ellipses
[1;32m      6[0m [38;5;28;01mfrom[39;00m [38;5;21;01mdynamax[39;00m[38;5;21;01m.[39;00m[38;5;21;01mnonlinear_gaussian_ssm[39;00m[38;5;21;01m.[39;00m[38;5;21;01mnonlinear_gaussian_ssm[39;00m [38;5;28;01mimport[39;00m ParamsNLGSSM, NonlinearGaussianSSM
[1;32m      7[0m [38;5;28;01mfrom[39;00m [38;5;21;01mdynamax[39;00m[38;5;21;01m.[39;00m[38;5;21;01mnonlinear_gaussian_ssm[39;00m[38;5;21;01m.[39;00m[38;5;21;01mextended_kalman_filter[39;00m [38;5;28;01mimport[39;00m extended_kalman_filter [38;5;28;01mas[39;00m ekf

[0;31mModuleNotFoundError[0m: No module named 'dynamax.plotting'
ModuleNotFoundError: No module named 'dynamax.plotting'

